src/main/java/com/opentok/MuteAllProperties.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * OpenTok Java SDK
+ * Copyright (C) 2021 Vonage.
+ * http://www.tokbox.com
+ *
+ * Licensed under The MIT License (MIT). See LICENSE file for more information.
+ */
+package com.opentok;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Defines values for the <code>properties</code> parameter of the
+ * {@link OpenTok#forceMuteAll(String, MuteAllProperties)} method.
+ *
+ * @see OpenTok#forceMuteAll(String, MuteAllProperties)
+ */
+public class MuteAllProperties {
+    private List<String> excludedStreams;
+
+    private MuteAllProperties(MuteAllProperties.Builder builder) {
+        this.excludedStreams = builder.excludedStreams;
+    }
+
+    /**
+     * Use this class to create a MuteAllProperties object.
+     *
+     * @see MuteAllProperties
+     */
+    public static class Builder {
+        private List<String> excludedStreams = new ArrayList<>();
+
+        /**
+         * Call this method to add a list of strings to the excludedStreams list
+         *
+         * @param ids A List of type {@link String}.
+         *
+         * @return The MuteAllProperties.Builder object with excludedStreams list.
+         */
+        public MuteAllProperties.Builder excludedStreamIds(List<String> ids) {
+            this.excludedStreams.addAll(ids);
+            return this;
+        }
+
+        /**
+         * Call this method to add a list of Stream to the excludedStreams list
+         *
+         * @param streams A List of type {@link Stream}.
+         *
+         * @return The MuteAllProperties.Builder object with excludedStreams list.
+         */
+        public MuteAllProperties.Builder excludedStreams(List<Stream> streams) {
+            this.excludedStreams.addAll(streams.stream().map(stream -> stream.getId()).collect(Collectors.toList()));
+            return this;
+        }
```

Comment:
I don't think we need this method. I think just including the `MuteAllProperties.Builder excludedStreamIds(List<String> ids)` method suffices.